crates/turbopack-css/src/references/internal.rs

```diff
@@ -0,0 +1,46 @@
+use anyhow::Result;
+use turbo_tasks::{primitives::StringVc, ValueToString, ValueToStringVc};
+use turbopack_core::{
+    asset::{Asset, AssetVc},
+    chunk::{ChunkableAssetReference, ChunkableAssetReferenceVc},
+    reference::{AssetReference, AssetReferenceVc},
+    resolve::{ResolveResult, ResolveResultVc},
+};
+
+/// A reference to an internal CSS asset.
+#[turbo_tasks::value]
+#[derive(Hash, Debug)]
+pub struct InternalCssAssetReference {
+    asset: AssetVc,
```

Comment:
```suggestion
    source: AssetVc,
    context: AssetContextVc,
```

This should lazily process the source asset

Reply:
The asset will already have been processed at that point, as we need to process it to compute other references.